src/js/state/QueryLibrary/types.ts

```diff
@@ -0,0 +1,50 @@
+export type QueryLibraryState = Group
+
+export interface Query {
+  id: string
+  name: string
+  value: string
```

Comment:
What do you think about naming this field `zql` instead of `value`?

Reply:
Sure, that works